recipes/syned/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "syned" %}
+{% set version = "1.0.12" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5c07ef5fb1212af0702c1b66bd0390e5f7938637e8f6a8c2041b9bf8c7e2128e
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: python
+
+requirements:
+  host:
+    - numpy
```

Comment:
Are these all needed? Normally `python` and `pip` is enough (and maybe `setuptools`)